src/test/java/kostka/moviecatalog/service/DbMovieServiceTest.java

```diff
@@ -16,22 +17,22 @@
 @RunWith(SpringRunner.class)
 @SpringBootTest(classes = MovieCatalogApplication.class)
 @Transactional
-public class MovieServiceImplTest {
+public class DbMovieServiceTest {
     @Autowired
     private MovieRepository movieRepository;
     @Autowired
-    private MovieServiceImpl movieServiceImpl;
+    private DbMovieService dbMovieService;
 
     @Test
     public void createMovieTest() {
-        movieServiceImpl.createMovie("test");
+        dbMovieService.createMovie(new MovieDto());
 
         Assertions.assertEquals(1, movieRepository.findAll().size());
-        Assertions.assertTrue(movieRepository.count() == 1);
+        Assertions.assertEquals(1, movieRepository.count());
```

Comment:
Tip/Optional: Jen v ramci education, klidne muzes jeste mrknout na assertJ knihovnu. Obsahuje lip citelne fluent api pro ty asserty a je pouzivana i na eOCS.

Reply:
Používá se tam i hamcrest na nějaké testy?

Reply:
Mám vytvořenou issue na test coverage tajze testy mne teprv čekají... Já vím, není to tdd...

Reply:
Hamcrest tam je taky, ale poslední "doporučení" od Daniela jsem dostal na AssertJ. Co jsem si všim v jeho MR, tak to taky používá.

TDD bych zase tolik neřešil na takovém výukovém projektu :) Já taky nepoužívám čisté TDD hned na začátku, když začínám něco psát :)